tools/benchmark/src/main/java/com/datadog/benchmark/internal/RequestBodyBuilder.kt

```diff
@@ -6,8 +6,7 @@
 
 package com.datadog.benchmark.internal
 
-import com.google.gson.JsonElement
-
 internal interface RequestBodyBuilder<T> {
-    fun buildJsonElement(data: List<T>): JsonElement
+
+    fun buildJson(data: List<T>): String
```

Comment:
to be precise, not return type is not `JsonXXX`, but simply `String`, so I guess method should be renamed to just `build`, because `String` return type doesn't imply that it should be serialized json there